src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelService.java

```diff
@@ -0,0 +1,199 @@
+    }
+
+    @Override
+    public void printChannel(Channel channel) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        System.out.printf("채널 단일 조회, 채널 이름: %s, 채널 내 유저 수 %d%n", channel.getChannelName(), channel.getUsers().size());
+        System.out.printf("채절 주인: '%s', 채널 내 유저 들=%s%n", channel.getHostUser().getUserName(), channel.getUsers());
+        //System.out.println(channel.toString());
+    }
+
+    @Override
+    public void printUsersFromChannel(Channel channel) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        System.out.printf("%s 채널 내 유저 검색, 채널 내 유저 수: %d%n",
+                channel.getChannelName(), channel.getUsers().size());
+
+        channel.getUsers().forEach(u -> System.out.println(u.getUserName()));
+    }
+
+    @Override
+    public void updateChannelName(User user, Channel channel, String newName) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        if(user.getStatus().equals(User.UserStatus.DEACTIVE)) {
+            System.out.printf("'%s'는 비활성 상태이므로 채널의 이름을 변경할 수 없습니다.%n", user.getUserName());
+            return; // 메서드 종료
+        }
+        if (channel.getHostUser().equals(user)) {
+            System.out.printf("'%s' 채널의 이름이 '%s' 으로 변경되었습니다.%n",
+                    channel.getChannelName(), newName);
+            channel.updateChannelName(newName);
+        } else {
+            System.out.printf("'%s' 은 '%s' 채널 주인이 아닙니다.%n",
+                    user.getUserName(), channel.getChannelName());
+        }
+    }
+
+    @Override
+    public void deleteChannel(User user, Channel channel) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        if(user.getStatus().equals(User.UserStatus.DEACTIVE)) {
+            System.out.printf("'%s'는 이미 비활성 상태입니다. 따라서 '%s' 채널을 삭제할 수 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return; // 메서드 종료
+        }
+        if(!channel.getUsers().contains(user)) {
+            System.out.printf("'%s'는 '%s' 채널에 존재하지 않아서 권한이 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return;
+        }
+
+        if (!user.equals(channel.getHostUser())) {
+            System.out.printf("'%s'는 '%s' 채널의 주인이 아니어서 지울 수 있는 권한이 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return;
+        }
+
+        System.out.printf("채널 삭제: %s%n", channel.getChannelName());
+        channel.getMessages()
+                .stream()
+                .map(message -> {
+                    message.getUser().removeMessage(message);
+                    return message;
+                });
+
+        channel.getUsers()
+                .stream()
+                .map(u -> {
+                    u.removeChannel(channel);
+                    return u;
```

Comment:
```suggestion
                .map(user -> {
                    user.removeChannel(channel);
                    return user;
```
가독성을 위해 약어 표현을 삼가해주세요.